packages/paste-website/src/pages/content/product-style-guide.mdx

```diff
@@ -192,56 +198,106 @@ Coming in the next release
+- Only **bold the most important information the user needs to retain** and deploy it only if user testing flags a need. 
+- Consider if putting an important sentence on a new line will do a better job of making it stand out than making it bold. 
+
+##### Italics
+**Avoid** using italic type wherever possible. 
+- If you need to draw attention to a word, use bold type.
+- If you need to quote a phrase, use quotation marks "like this." 
 
 #### Capitalization
 **Do** capitalize proper nouns – products, people, places.
-- If a brand or product name uses a mix of upper and lower case (TwiML, eBay) use the name's convention.
 - Products include features, programs, or systems that Twilio has given a specific name to.
+- If a brand or product name uses a mix of upper and lower case (TwiML, eBay) use the name's convention.
 - People are named individuals or identifiable roles.
   - *Kris Brunner, Chief Engineer*
-  - *Contact your System Administrator*
 
-**Don't** capitalize general terms (*email, contact center, platform*)
+**Don't** capitalize general terms (*email, contact center, platform, phone number*)
 - Tip: If it's something Twilio has created or owns, it takes a capital.
 
 #### Code snippets
 Coming in the next release
 
-#### Dates
+#### Currency
 Coming in the next release
 
+#### Dates
+**Do** use American formatting as your default way to represent dates.
+- *MM/DD/YYYY* 
+- *Day, Month DD, YYYY* e.g., Sunday, May 17, 2020
+- A leading zero isn't needed for single digit dates, *3/12/2019* not *03/12/2020*
+
+**Do** abbreviate months to their first 3 letters if space is limited.
+- *Jan, Feb, Mar*
+
 #### I/You/We/Us
 **Do** refer to the user as *you* and to Twilio as *we*.
 - Tip: If the user has agency or ownership, use ‘I’ or ‘my’.
   - *If you still can’t connect, contact our support team.*
   - *I understand.* (This usage is almost exclusively for consent.)
   - *To access billing information, select My Account.*
 
+**Do** say *we* instead of *Twilio*
+- It's us talking, so there's no need to refer to us in the third person.
+- If "Twilio" is included in a product or feature name, that's ok as that's using it as a noun.
+
 #### Lists (all)
 **Do** start each list item with the same part of speech – usually a noun or verb – to make the list "parallel."
 - Punctuate the end of each item with a period if it is a complete sentence.
 - Be consistent within the list. If one item takes a full point, all items should.
 
-**Don't** punctuate the end of an item in a vertical list with a comma or elipsis "..."
+**Don't** punctuate the end of an item in a vertical list with a comma or ellipsis "..."
 
 #### Lists (bullet points)
-- **Do** use bulleted lists for items linked by a topic. Make the list topic clear in the sentence or heading that precedes it.
+**Do** use bulleted lists for items linked by a topic. Make the list topic clear in the sentence or heading that precedes it.
```

Comment:
Any guidance here on what style the list should be or maybe which styles to avoid?

Reply:
I'd defer to /components/list on that front. From a content perspective it doesn't matter hugely, as long as the hierarchy that's detailed in the Paste guidelines is adhered to. 

Was there something specific you had in mind?